### PR TITLE
Update README.md with Proxy support information.

### PR DESCRIPTION
**Pull request by Mike97M**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ bot = InstaBot('login', 'password',
                 max_like_for_one_tag = 5,
                 unfollow_break_min = 15,
                 unfollow_break_max = 30,
-                log_mod = 0)
+                log_mod = 0
```
Comment (LevPasha) [inline README.md]: Miss ,

File: README.md [hunk 2]
```diff
@@ -51,7 +52,7 @@ bot = InstaBot('login', 'password',
 | unfollow_break_min   | int | Minimum seconds for unfollow break pause             | 15                               | 30                                              |
 | unfollow_break_max   | int | Maximum seconds for unfollow break pause             | 30                               | 60                                              |
 | log_mod              | int | logging mod                                          | 0                                | 0 log to console, 1 log to file, 2 no log.      |
-
+| proxy             | string | Access instagram through a proxy server                                          |                                | Without authentication: proxy:port, example: 10.10.1.10:3128, with authentication: user:pass@proxy:port, example: mike:d23dedf@10.10.1.10:3128 |
```
Comment (LevPasha) [inline README.md]: Use something like user:password@10.10.10.10:1234

**Comment by Mike97M** [comment]
Done

**Comment by LevPasha** [comment]
miss "," after log_mod = 0

**Comment by Mike97M** [comment]
Done

**Pull request merged**